src/lib/components/avatar.svelte

```diff
@@ -16,10 +22,11 @@
 	tabindex={0}
 >
 	<div class="img" style={`height: ${size}px;`}>
-		{#if picture}
+		{#if picture && isCID(picture)}
```

Comment:
Personally I would use another prop for the so that we don't need to check for CID. So if the Avatar has `picture` defined, show that if not, show `dicebear` with `dicebearSeed` prop. So we would use it as `<Avatar picture={chat.picture} dicebearSeed={chat.id} />

But I am fine with this implementation as well, just wanted to check if you considered this.

Reply:
Something is missing from your comment and I don't understand what exactly you are saying here :smile:

Reply:
Ah weird... Anyways the gist was if you considered to not pass the `chat.id` for dicebear as `picture` but instead as separat props. This way there is no need to check if it `isCID` and it serves as fallback.

`<Avatar picture={chat.picture} dicebearSeed={chat.id}`

Reply:
Okay, I added a `seed` prop because the code is better without the conditionals everywhere.